package.json

```diff
@@ -131,7 +131,7 @@
     "lint": "eslint --max-warnings 0 --config .eslintrc .",
     "lint:fix": "eslint --max-warnings 5 --config .eslintrc . --fix",
     "postpack": "rimraf oclif.manifest.json",
-    "prepack": "rimraf lib && tsc -b && oclif manifest && oclif readme",
+    "prepack": "npm run build && oclif manifest && oclif readme",
     "prepublishOnly": "npm run build",
```

Comment:
do we need this if we build it in `prepack` stage?

Reply:
Yeah since we are building in `prepublish` stage which is run before prepack we don't need this. Though we are also running `oclif readme` in `version` so we might now need that too 

https://github.com/asyncapi/cli/blob/9fecc14b1f47ae4df238d9ef9cc4778a869d9e5c/package.json#L140

also I think this is why the TOC is not working

Reply:
imho:

- we should only have `prepublishOnly` that runs: `npm run build && oclif manifest`
- `version` script is not needed, can be removed, same `prepack`. Same with `readme` script I guess
- `oclif readme` should be added to `generate:assets`. Although I do not really know what `oclif readme` is actually doing 😆 But if it is doing some important update to readme, it should be in `generate:assets` script. [This workflow](https://github.com/asyncapi/cli/blob/master/.github/workflows/if-nodejs-version-bump.yml#L31) will make sure readme is updated in GH repo. We can also run `generate:assets` in `prepublishOnly` script if we need also some magic updates in readme to show up in NPM package readme view

makes sense?

Reply:
I have removed `oclif readme`. It creates documentation which is handled here https://github.com/asyncapi/cli/issues/123